src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,115 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size;
+    private Object[] arrayList;
+
+    public ArrayList() {
+        arrayList = new Object[DEFAULT_CAPACITY];
+    }
+
+    private String errorOutOfBoundsMsg(int index) {
```

Comment:
```suggestion
    private String getErrorOutOfBoundsMessage(int index) {
```

please move private methods below public

Reply:
![image](https://user-images.githubusercontent.com/22300600/121507232-1e6faf00-c9ed-11eb-8d68-274166accf6d.png)